Extension/lib/pages/options.js

```diff
@@ -281,7 +281,7 @@ PageController.prototype = {
     _showFilterLoader: function (antiBannerFilter) {
         var el = $("input[name='filterId'][value='" + antiBannerFilter.filterId + "']").closest(".s-page-table-row");
         var acceptableAdsFilter = antiBannerFilter.filterId == AntiBannerFiltersId.ACCEPTABLE_ADS_FILTER_ID; // Don't render for acceptable ads filter
-        if (el.length == 0 && !acceptableAdsFilter) {
+        if (el.length == 0 && antiBannerFilter.enabled && !acceptableAdsFilter) {

```

Comment:
If filter element is visible, a loader gonna be displayed. But it will not be added to the list if it is not in it, otherwise all filters will be added and shown in the options page.